frontends/web/src/components/devices/bitbox02bootloader/toggleshowfirmwarehash.tsx

```diff
@@ -14,48 +15,41 @@
  * limitations under the License.
  */
 
-import { ChangeEvent, useState } from 'react';
+import { ChangeEvent, useEffect, useState } from 'react';
 import { useTranslation } from 'react-i18next';
-import { load } from '../../../decorators/load';
-import { apiPost } from '../../../utils/request';
+import { useLoad } from '../../../hooks/api';
+import { getShowFirmwareHash, setShowFirmwareHash } from '../../../api/bitbox02';
 import { Toggle } from '../../toggle/toggle';
 
-type TToggleProps = {
-    deviceID: string;
-}
-
-type TLoadedProps = {
-    enabled: boolean;
-}
-
-type TProps = TToggleProps & TLoadedProps;
+type Props = {
+  deviceID: string;
+};
 
-const ToggleFWHash = ({ enabled, deviceID }: TProps) => {
+export const ToggleShowFirmwareHash = ({ deviceID }: Props) => {
   const { t } = useTranslation();
-  const [enabledState, setEnabledState] = useState<boolean>(enabled);
+  const [enabledState, setEnabledState] = useState<boolean>(false);
```

Comment:
changed initial state to false, as on mount enabledConfig is still undefined until getShowFirmwareHash loads